### ISSUE TITLE
Mind Control effects break if you move too quickly after it takes effect

### ISSUE THREAD

**Issue by edgeucated** [TeamEverlook/Everlook-BugTracker#250]
**Describe the bug**
When you enter a Mind control effect such a priest mind control/Orb of Domination/Elixir of Dream Vision if you move immediately after gaining the effect the effect your main character will move and break the mind control effect 

**Quest/Item/Spell Link**
Elixir of Dream Vision
https://classicdb.ch/?item=9197
Orb of Domination
https://classicdb.ch/?object=177808
Priest Mind Control
https://classicdb.ch/?spell=10912

**To Reproduce**
Use a mind control effect like Dream Vision Elixir, Orb of Domination, or Priest mind control and move immediately after gaining the effect.  The mind control effect will often break right away

**Expected behavior**
Mind control effects should no break even when moving right after gaining the effect.

**Comment by red-candle** [comment]
Yeah, this goes for hunters controlling their pet too - there's a delay between executing the cast and the effect being applied (for some reason).
